cmd/kube-dns/app/options/options.go

```diff
@@ -138,6 +140,9 @@ func (s *KubeDNSConfig) AddFlags(fs *pflag.FlagSet) {
 	fs.Var(clusterDomainVar{&s.ClusterDomain}, "domain",
 		"domain under which to create names")
 
+	fs.StringVar(&s.NameServers, "nameservers", s.NameServers,
+		"List of ip:port, separated by commas of recursive nameservers to forward queries to.")
```

Comment:
The `:port` is optional, is it not?

Remove `recursive` (upstream may not necessarily be such)

Add "If set, overrides upstream servers taken from the `nameserver` option in `/etc/resolv.conf`.

Give an example of a valid value in the description: "Example: 8.8.8.8,.8.8.4.4"